src/angular-app/languageforge/lexicon/settings/configuration/field-unified-view.model.ts

```diff
@@ -135,10 +165,10 @@ export class ConfigurationFieldUnifiedViewModel {
     }
   }
 
-  static selectAllGroupColumn(settings: SettingsBase[], selectAll: SettingsBase, groupIndex: number): void {
+  static selectAllGroupColumn(settings: FieldSettings[], selectAll: SettingsBase, groupIndex: number): void {
```

Comment:
You can't change SettingsBase here, as this function is also called in the input systems settings portion of this view ([here](https://github.com/sillsdev/web-languageforge/blob/59dcc8b2a1fd41f84d47dea1fbfcf729c5bb444f/src/angular-app/languageforge/lexicon/settings/configuration/configuration-fields.component.html#L63)). When it receives input systems settings, there's no `fieldName` property on the settings object; instead there's a `tag` property containing "en" for English, etc.

So what you'll need to do is check for the presence of a `fieldName` property, and if it's present, cast the SettingsBase object as FieldSettings before passing it to checkIfAllEntryFieldsRowSelected.

Reply:
good catch @rmunn